app/models/hmcts_common_platform/hearing_summary.rb

```diff
@@ -25,5 +23,19 @@ def hearing_days
     def court_centre
       HmctsCommonPlatform::CourtCentre.new(data[:courtCentre])
```

Comment:
Is this now overkill to create this object just to pass the id to `HmctsCommonPlatform::Reference::CourtCentre`?


How would you feel about
```
  def hmcts_common_platform_reference_court_centre
      @hmcts_common_platform_reference_court_centre ||= HmctsCommonPlatform::Reference::CourtCentre.find(data.dig(:courtCentre, :id)
    end
```